### ISSUE TITLE
Extension issue

### ISSUE THREAD

**Issue by Gcsilva** [DEVSENSE/phptools-docs#147]
- Issue Type: `Bug`
- Extension Name: `phptools-vscode`
- Extension Version: `1.10.9721`
- OS Version: `Windows_NT x64 10.0.19042`
- VS Code version: `1.68.1`

:warning: We have written the needed data into your clipboard. Please paste! :warning:

**Comment by Gcsilva** [comment]
```json 
{
	"messages": [],
	"activationTimes": {
		"codeLoadingTime": 704,
		"activateCallTime": 16,
		"activateResolvedTime": 1071,
		"activationReason": {
			"startup": false,
			"extensionId": {
				"value": "DEVSENSE.phptools-vscode",
				"_lower": "devsense.phptools-vscode"
			},
			"activationEvent": "onLanguage:php"
		}
	},
	"runtimeErrors": [
		{
			"name": "ReferenceError",
			"message": "Cannot access 'i' before initialization"
		}
	],
	"runningLocation": {
		"affinity": 0,
		"kind": 1
	}
}
```

**Comment by jakubmisek** [comment]
thank you for reporting the issue.

We "think" what is the problem, so we'll try to fix this within the next update. You can install a Pre-Release version of PHP Tools once we release it (ETA within 24 hours).

**Comment by jakubmisek** [comment]
do you have any other information? There should be a stack trace, eventually, the complete error details would be listed in `Help` / `Toggle Developer Tools` / `Console`. 

thank you

**Comment by Gcsilva** [comment]
Thank you for your quick answer.

Below is the stack trace below:

```
console.ts:137 [Extension Host] stack trace: ReferenceError: Cannot access 'i' before initialization	at c:\Users\araujoand\.vscode\extensions\devsense.phptools-vscode-1.10.9721-win32-x64\out\src\extension.js:1:6708	at Generator.next (<anonymous>)	at c:\Users\araujoand\.vscode\extensions\devsense.phptools-vscode-1.10.9721-win32-x64\out\src\extension.js:1:374	at new Promise (<anonymous>)	at f (c:\Users\araujoand\.vscode\extensions\devsense.phptools-vscode-1.10.9721-win32-x64\out\src\extension.js:1:125)	at c:\Users\araujoand\.vscode\extensions\devsense.phptools-vscode-1.10.9721-win32-x64\out\src\extension.js:1:5051	at c:\Users\araujoand\.vscode\extensions\devsense.phptools-vscode-1.10.9721-win32-x64\out\src\extension.js:1:6979	at Generator.next (<anonymous>)	at o (c:\Users\araujoand\.vscode\extensions\devsense.phptools-vscode-1.10.9721-win32-x64\out\src\extension.js:1:179)	at processTicksAndRejections (node:internal/process/task_queues:96:5)
I @ console.ts:137
mainThreadExtensionService.ts:80 ReferenceError: Cannot access 'i' before initialization
	at c:\Users\araujoand\.vscode\extensions\devsense.phptools-vscode-1.10.9721-win32-x64\out\src\extension.js:1:6708
	at Generator.next (<anonymous>)
	at c:\Users\araujoand\.vscode\extensions\devsense.phptools-vscode-1.10.9721-win32-x64\out\src\extension.js:1:374
	at new Promise (<anonymous>)
	at f (c:\Users\araujoand\.vscode\extensions\devsense.phptools-vscode-1.10.9721-win32-x64\out\src\extension.js:1:125)
	at c:\Users\araujoand\.vscode\extensions\devsense.phptools-vscode-1.10.9721-win32-x64\out\src\extension.js:1:5051
	at c:\Users\araujoand\.vscode\extensions\devsense.phptools-vscode-1.10.9721-win32-x64\out\src\extension.js:1:6979
	at Generator.next (<anonymous>)
	at o (c:\Users\araujoand\.vscode\extensions\devsense.phptools-vscode-1.10.9721-win32-x64\out\src\extension.js:1:179)
	at processTicksAndRejections (node:internal/process/task_queues:96:5)
$onExtensionRuntimeError @ mainThreadExtensionService.ts:80
```

**Comment by ctf0** [comment]
same here as well

```
Version: 1.69.0-insider
Commit: 0e3304bca9e862eddc7e437e2bab8a4a4a4f33c9
Date: 2022-06-29T05:18:24.550Z
Electron: 18.3.5
Chromium: 100.0.4896.160
Node.js: 16.13.2
V8: 10.0.139.17-electron.0
OS: Darwin arm64 21.5.0
```
```js
console.ts:137 [Extension Host] rejected promise not handled within 1 second: ReferenceError: Cannot access 'i' before initialization (at console.<anonymous> (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:85:93779))
D @ console.ts:137
console.ts:137 [Extension Host] stack trace: ReferenceError: Cannot access 'i' before initialization
    at /Users/ctf0/.vscode-insiders/extensions/devsense.phptools-vscode-1.10.9721-darwin-arm64/out/src/extension.js:1:6646
    at Generator.next (<anonymous>)
    at /Users/ctf0/.vscode-insiders/extensions/devsense.phptools-vscode-1.10.9721-darwin-arm64/out/src/extension.js:1:312
    at new Promise (<anonymous>)
    at f (/Users/ctf0/.vscode-insiders/extensions/devsense.phptools-vscode-1.10.9721-darwin-arm64/out/src/extension.js:1:63)
    at /Users/ctf0/.vscode-insiders/extensions/devsense.phptools-vscode-1.10.9721-darwin-arm64/out/src/extension.js:1:4989
    at /Users/ctf0/.vscode-insiders/extensions/devsense.phptools-vscode-1.10.9721-darwin-arm64/out/src/extension.js:1:6917
    at Generator.next (<anonymous>)
    at o (/Users/ctf0/.vscode-insiders/extensions/devsense.phptools-vscode-1.10.9721-darwin-arm64/out/src/extension.js:1:117)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) (at console.<anonymous> (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:85:93779))
```

**Comment by jakubmisek** [comment]
Thank you! We're preparing the fix.

**Comment by jakubmisek** [comment]
We have release a Pre-Release which you try now.

Preparing the full Release within today.

Thank you!

**Comment by jakubmisek** [comment]
fixed in `1.11`
